A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -1588,10 +1610,22 @@ var A320_Neo_UpperECAM;
         getN1GaugeExtraMessage() {
             if (Simplane.getEngineThrottle(this.index) < 0) {
                 return "REV";
+            } else if (this.timerAvail >= 0) {
+                return "AVAIL";
             } else {
                 return "";
             }
         }
+        getEngineStartStatus() {
+            const engineId = this.index + 1;
+            const value = (SimVar.GetSimVarValue("GENERAL ENG STARTER:" + engineId, "bool") === 1) ? true : false;
```

Comment:
There's no need for this ternary, the condition is already producing a boolean value.